.github/workflows/sync.yml

```diff
@@ -0,0 +1,92 @@
+name: Sync
+
+on:
+  workflow_dispatch:
+
+jobs:
+  sync:
+    strategy:
+      matrix:
+        go-version: [1.19.x]
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Get current date
+      id: date
+      run: echo "date=$(date +'%Y-%m-%d')" >> "$GITHUB_OUTPUT"
+    - name: Checkout code
+      uses: actions/checkout@v3
+    - name: GitHub user
+      run: |
+        # https://api.github.com/users/github-actions[bot]
+        git config user.name 'github-actions[bot]'
+        git config user.email '41898282+github-actions[bot]@users.noreply.github.com'
+    - name: Fetch latest spec to oas.fetched/
+      id: fetch
+      run: |
+        make fetch
+        git add oas3.fetched
+        echo `git commit -m 'sync: fetch ${{ steps.date.outputs.date }} spec and apply patches'`
+    - name: Build single-file spec in oas3.stitched/metal_openapi.yaml
```

Comment:
We removed this step in `metal-go` (and it turned out that combining the spec might have introduced a bug in how `include` and `exclude` were handled).  Removing it now might introduce some minor code changes & make this PR bigger, so if you'd prefer to keep it as-is and remove the stitch step later, that's fine.